components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -769,10 +754,17 @@
         var userSelection = this._authenticationLabel.replace("__serverID__", this._selectedServer);
         $$('#user-selection h2')[0].innerHTML = userSelection;
         var userSelectionHint = this._authenticationHint.replace(/__serverID__/g, this._selectedServer);
-        $$('#user-selection p')[0].innerHTML = userSelectionHint;
+        $('user-selection-intro').innerHTML = userSelectionHint;
+        $('user-selection-intro').show();
         var groupSelection = this._remoteGroupLabel.replace("__serverID__", this._selectedServer);
         $$('#group-selection h2')[0].innerHTML = groupSelection;
-
+        if (serverData.registrationURL != "" &amp;&amp; !this._lastApprovedUser) {
+           var remoteRegistration = this._remoteRegistrationHint.replace("__serverID__", this._selectedServer).replace("__registrationUrl__", "&lt;a target=\"_blank\" href=\""+serverData.registrationURL+"\"&gt;"+serverData.registrationURL+"&lt;/a&gt;");
```

Comment:
The remoteURL is being altered a few lines above in the `_populateServerList` function, to prepend `http://` (using the `_addHTTP`) function. I would do something similar for the registration url (if the admin entrers one url without its protocol, they're very likely to do the same for the other).

However, I would account for the fact that in some cases the signing up can be done by contacting an email address ,
 since not all PhenoTips installations that receive contributions will have registration open, in which case the prefix should be `mailto:` instead.

Moreover, imo this cleaning up of the url should be done serverside in the service, rather than in javascript. 

@sdumitriu @veronikaslc thoughts?